vignettes/data-combined-plotting.Rmd

```diff
@@ -0,0 +1,182 @@
+---
+title: "Visualizations with `DataCombined`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Visualizations with `DataCombined`}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  warning = FALSE,
+  message = FALSE,
+  dpi = 300,
+  comment = "#>",
+  out.width = "100%",
+  fig.height = 6,
+  fig.width = 8
+)
+```
+
+## Introduction
+
+You have already seen how the `DataCombined` class can be utilized to store observed and/or simulated data (if not, read [Working with `DataCombined` class](data-combined.html)).
+
+Let's first create a `DataCombined` object, which we will use to demonstrate different visualizations available.
+
+```{r}
+library(ospsuite)
+
+# simulated data
+simFilePath <- system.file("extdata", "Aciclovir.pkml", package = "ospsuite")
+sim <- loadSimulation(simFilePath)
+simResults <- runSimulation(sim)
+outputPath <- "Organism|PeripheralVenousBlood|Aciclovir|Plasma (Peripheral Venous Blood)"
+
+# observed data
+obsData <- lapply(
+  c("ObsDataAciclovir_1.pkml", "ObsDataAciclovir_2.pkml", "ObsDataAciclovir_3.pkml"),
+  function(x) loadDataSetFromPKML(system.file("extdata", x, package = "ospsuite"))
+)
+names(obsData) <- lapply(obsData, function(x) x$name)
```

Comment:
I thought `observeddata$name` will be taken as per default if no `names ` argument is provided in `addDataSets()`?

Reply:
This line is only for assigning names to the list `obsData` for convenient access... nothing to do with `DataCombined` :)

Reply:
ok